tools/test/test_depsolve.py

```diff
@@ -13,6 +13,10 @@
     "./test/data/testrepos/custom/",
 ]
 
+RELEASEVER = "9"
```

Comment:
Nice one! This was one of my nitpicks ;)

Reply:
On a long enough timeline, your nitpicks per PR will trend to zero.  That's my goal!